compliance_checker/cf/util.py

```diff
@@ -403,21 +418,22 @@ def compare_unit_types(specified, reference):
     msgs = []
     err_flag = False
     try:
-        specified_unit = Unit(specified)
-    except ValueError:
+        specified_unit = ut_system.unit(specified)
+    except UnresolvableUnitException:
         msgs.append(f"Specified conversion unit f{specified} may not be valid UDUnits")
         err_flag = True
 
     try:
-        reference_unit = Unit(reference)
-    except ValueError:
+        reference_unit = ut_system.unit(reference)
+    except UnresolvableUnitException:
         msgs.append(f"Specified conversion unit f{reference} may not be valid UDUnits")
         err_flag = True
 
     if err_flag:
         return msgs
 
-    unit_convertible = specified_unit.is_convertible(reference_unit)
+    # FIXME: This one passed with the wrong syntax!! Our tests are probably not covering this!!!
```

Comment:
Is this something on the `pyudunits2` side?

Reply:
Nope. It is a bug in compliance-checker. It did not fail when it should have.( I'm taking notes in this draft PR b/c I no longer can trust my brain to remember things.)

Reply:
What was the issue here?

Reply:
I didn't see it either, until I remembered that I called the method `is_convertible_to` https://github.com/pelson/pyudunits2/blob/f75f1afd27d4b66ef77fc9db9301ae914111d288/pyudunits2/_unit.py#L394

Reply:
See #1119